### PR TITLE
WAR-1024: JUNIT file change: Added title for Project/Suite/case, Added description for step

### PR DESCRIPTION
**Pull request by Annam16**
1.Added Project/Suite/Case/Step descriptions in JUNIT result file as project_description/suite_description/testcase_description/step_description.
2. Run any project/suite/case to check this feature or cli_regression results which is attached in JIRA ticket.

### CODE REVIEW

**Comment by kaheichan** [comment]
Looks like there is still merge conflict @Annam16 , can you please fix that?

File: warrior/WarriorCore/project_driver.py [hunk 1]
```diff
@@ -290,11 +315,72 @@ def execute_project(project_filepath, auto_defects, jiraproj, res_startdir, logs
...
-                                                 project_error_value)
+        runmode, value = common_execution_utils.get_runmode_from_xmlfile(testsuite)
+        retry_type, retry_cond, retry_cond_value, retry_value,\
+            retry_interval = common_execution_utils.get_retry_from_xmlfile(testsuite)
+        if runmode is not None:
+            if testsuite.find("runmode") is not None and\
+              testsuite.find("runmode").get("attempt") is not None:
+                print_info("runmode attempt: {0}".format(testsuite.find("runmode").get("attempt")))
+            # if runmode is 'ruf' & step_status is False, skip the repeated
+            # execution of same TC step and move to next actual step
```
Comment (kaheichan) [inline warrior/WarriorCore/project_driver.py]: Please update the documentation, this should be testsuite

**Comment by Annam16** [comment]
Hi @kaheichan Modified the documentation in project_driver.py

**Comment by Annam16** [comment]
@kaheichan resolved merge conflicts. Please approve and merge.

**Pull request merged**